.env

```diff
@@ -0,0 +1 @@
+YELP_API_KEY=6Ialhkd_qwKhYNYw38mRefedsZqtjrJFYf_rRZsmSE-7iDVEtuez6PYif6MRUhz5ZEGVi7ffnExj9JObNmGEtqCRutd8xfdZAad3YUcAmiBcnmIUiIiOsOYgUaB8YHYx
```

Comment:
You may want to look into using Github Secrets to obscure your public API key on Github

Reply:
This is actually a convo we should have w Wilson - what's the best way to grant API access to all users of the app? Should I use my API key (and hide it w gh secrets), or have users get their own keys?